commit 329a53bef199dcae3f543e6f01130c68452c3bce
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Thu Jan 24 16:36:19 2019 +0000

    staging: bcm2835-codec: Fix potentially uninitialised vars
    
    src_m2m_buf and dst_m2m_buf were printed in log messages
    when there are code paths that don't initialise them.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
index 4ef16379f216..c26760faae2d 100644
--- a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
+++ b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
@@ -743,7 +743,7 @@ static void device_run(void *priv)
 	struct bcm2835_codec_ctx *ctx = priv;
 	struct bcm2835_codec_dev *dev = ctx->dev;
 	struct vb2_v4l2_buffer *src_buf, *dst_buf;
-	struct m2m_mmal_buffer *src_m2m_buf, *dst_m2m_buf;
+	struct m2m_mmal_buffer *src_m2m_buf = NULL, *dst_m2m_buf = NULL;
 	struct v4l2_m2m_buffer *m2m;
 	int ret;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1804_329a53bef199dcae3f543e6f01130c68452c3bce.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1804_329a53bef199dcae3f543e6f01130c68452c3bce.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
